modules/migration-mtc-release-notes-1-8.adoc

```diff
@@ -0,0 +1,73 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8_{context}"]
+= {mtc-full} 1.8 release notes
+
+[id="new-features-and-enhancements-1-8{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+This release has no new features and enhancements.
+
```

Comment:
Suggestion: Comment out or delete line 12 since the release has no new features.

Reply:
Good call

Reply:
@RichardHoch thanks